<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="MyClientGB">
          <property name="title">
           <string>MyClient</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="IPL">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
                 </property>
                 <property name="text">
                  <string>IP</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="IPLE"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="PortL">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Port</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="PortLE"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="ConnectPB">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
               </property>
               <property name="text">
                <string>Connect</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="DisconnectPB">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);</string>
               </property>
               <property name="text">
                <string>Disconnect</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="MessageL">
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
               </property>
               <property name="text">
                <string>Enter the Message</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="MessageLE"/>
             </item>
             <item>
              <widget class="QPushButton" name="SendPB">
               <property name="text">
                <string>Send</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ConsoleDataGB">
          <property name="title">
           <string>Console Data</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QListWidget" name="ConsoleLW"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="ConsoleClearPB">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="ReceivedGB">
        <property name="title">
         <string>Received Data</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QListWidget" name="ReceivedLW"/>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="ReceivedClearPB">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
